bob/src/cluster/quorum.rs

```diff
@@ -239,6 +246,51 @@ impl Quorum {
             Ok(())
         }
     }
+
+    pub(crate) async fn delete_aliens(
```

Comment:
There are problems with the deletion logic in aliens. If there is an entry in the aliens, then after deletion it will continue to be available for read. This is bad behavior. We need to run the deletion first on the target nodes, and then on the aliens of all nodes. It is important also, that on the nodes from `get_support_nodes`, the deletion in aliens should be forced (the deletion record is always made), and on the rest, it is made only if the data is presented. This will require to extend gRPC protocol with additional field (`force_delete`)

Reply:
Should be done now